.gitignore

```diff
@@ -1,5 +1,5 @@
 dist/
 dist-web/
 node_modules
-package-lock.json
```

Comment:
This could cause a problem if accidentally checked in since we don't maintain it. We've been maintaining yarn.lock instead.

Reply:
:+1: